scripts/godstime.js

```diff
@@ -0,0 +1,5 @@
+var name="Alade-Raji Far'hah";
+id = 'HNG-00449';
+lang = 'node.js';
+
+console.log('Hello World, this is', name,'with HNGi7 ID', id,'using',lang,'for stage 2 task');
```

Comment:
hey @Reehah, somethings to look at for review. add 'var' to all your variable before you initialize them for consistency. also, add a variable for your email and incorporate it into your final output. 
The output should follow this; 
**Hello World, this is [full name] with HNGi7 ID [ID] and email [email] using [language] for stage 2 task**
ps: everything between the stars is what you want to output the console.

With the above edits all should be well.